suse2013/common/trim-verbatim.xsl

```diff
@@ -12,61 +12,97 @@
   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" >
 
 <xsl:template match="programlisting/text()|screen/text()|synopsis/text()">
```

Comment:
Look at this XML snippet:

```xml
<screen>
<emphasis>bla</emphasis>
</screen>
```

The tree looks like this:

```
           screen
+------------+------------+
text()     emphasis    text()
             |
           text()
```

That means, you have **two** separate text nodes. If you have more elements inside `<screen>` you have more text nodes in between of course.

Not sure if you have thought about this, but just to mention it.